mkosi/archive.py

```diff
@@ -63,6 +67,72 @@ def make_tar(src: Path, dst: Path, *, tools: Path = Path("/"), sandbox: Sequence
         )
 
 
+def make_oci(context: Context, dst: Path) -> None:
+    ca_store = dst / "blobs" / "sha256"
+    ca_store.mkdir(parents=True)
+
+    layer_archive = context.staging / context.config.output_tar
+    layer_digest = hash_file(layer_archive)
+    # Should we create a symlink at the old path such that the tar can still be used?
+    layer_archive.rename(ca_store / layer_digest)
+
+    oci_config = {
+        # TODO Add more metadata? Created timestamp, history, config...
+        "architecture": context.config.architecture.to_oci(),
+        "os": "linux",
```

Comment:
Is that always just "linux" or should that be the distro?

Reply:
That is always linux (https://github.com/opencontainers/image-spec/blob/365fa41472a19107b0b4cbc17e141a9b97f9b66e/config.md?plain=1#L108)

Reply:
What a terrible spec, they could link to whatever they reference. Could you add a comment explaining that this is (I suppose) the first half (i.e. non-arch) output of `go tool dist list`?

Reply:
They do (and it works in the rendered markdown) though I linked the raw markdown where the link target is in the footnotes: https://golang.org/doc/install/source#environment. I will add the comment though